src/Execution/ReportErrorHandler.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace Nuwave\Lighthouse\Execution;
+
+use Closure;
+use GraphQL\Error\Error;
+use Illuminate\Contracts\Debug\ExceptionHandler;
+
+/**
+ * Report errors through the default exception handler configured in Laravel.
+ */
+class ReportErrorHandler implements ErrorHandler
```

Comment:
```suggestion
class ReportingErrorHandler implements ErrorHandler
```

Could possibly be a more descriptive name for it.